clients/admin-ui/src/types/api/models/BasicSystemResponse.ts

```diff
@@ -114,6 +114,10 @@ export type BasicSystemResponse = {
    * The unique identifier for the vendor that's associated with this system.
    */
   vendor_id?: string;
+  /**
+   * If specified, the unique identifier for the vendor that was previously associated with this system.
+   */
+  previous_vendor_id?: string;
```

Comment:
@allisonking! I got to use the instructions you typed up for me here! https://github.com/ethyca/fides/pull/4235#discussion_r1349288757 🙏

Reply:
amazing!!!